specification/iotoperations/IoTOperations.Management/models/dataflows/dataflowEndpoints.tsp

```diff
@@ -191,6 +196,9 @@ union DataflowEndpointFabricPathType {
 
 @doc("Kafka endpoint properties")
 model DataflowEndpointKafka {
+  @doc("Authentication configuration. NOTE - only authentication property is allowed per entry.")
+  authentication?: DataFlowEndpointKafkaAuthentication;
```

Comment:
This means optional right? If the `authentication` is omitted then it's no-auth?

Reply:
I thought the agreement was to use the `authenticationMethod` enum to support no-auth? 

https://msazure.visualstudio.com/One/_workitems/edit/28622511

![image](https://github.com/user-attachments/assets/806cd855-650d-4d25-9df2-21859b248ccf)